todo/todo.py

```diff
@@ -28,151 +28,141 @@
+    " of tasks for you to do. It provides a simple listing,"
+    " adding, and completing for you to work with, and is highly"
+    " customizable so you get the best out of it"
+)
 _comic_link = "https://raw.githubusercontent.com/Just-Jojo/JojoCog-Assets/main/data/todo_comic.jpg"
-
-
-def positive_int(arg: str) -> int:
-    """Returns a positive int"""
-    try:
-        ret = int(arg)
-    except ValueError:
-        raise commands.BadArgument(f"{arg} is not a integer.")
-    if ret <= 0:
-        raise commands.BadArgument(f"{arg} is not a positive integer.")
-    return ret
+_no_todo_message = (
+    "You don't have any todos!" " Add one using `{prefix}todo add <todo>`!"
+)
+_no_completed_message = (
+    "You don't have any completed todos!"
+    " You can complete a todo using `{prefix}todo complete <indexes...>`!"
+)
 
 
 class ToDo(commands.Cog):
-    """A simple, highly customizable todo list for Discord"""
+    """A simple and highly customizable todo list for Discord"""
 
-    __version__ = "1.0.2"
-    __author__ = [
-        "Jojo",
-    ]
+    __version__ = "1.1.0"
+    __author__ = ["Jojo#7791"]
 
-    def __init__(self, bot):
+    def __init__(self, bot: Red):
         self.bot = bot
-        self.config = Config.get_conf(self, 19924714019, force_registration=True)
+        self.config = Config.get_conf(
+            cog_instance=self, identifier=19924714019, force_registration=True
+        )
         self.config.register_user(**_config_structure)
 
-    def format_help_for_context(self, ctx):
-        """Thankie thankie Sinbad"""
+    def format_help_for_context(self, ctx: commands.Context):
         return (
             f"{super().format_help_for_context(ctx)}"
-            f"\n\n__Current Version:__ `{self.__version__}`\n"
-            f"__Author:__ `{', '.join(self.__author__)}`"
+            f"\n\n**Current Version:** `{self.__version__}`"
+            f"\n**Author:** `{', '.join(self.__author__)}`"
         )
 
-    async def red_delete_data_for_user(
-        self,
-        requester: typing.Literal["discord", "owner", "user", "user_strict"],
-        user_id: int,
-    ) -> None:
-        await self.config.user_from_id(user_id).clear()
-
     ### Setting commands ###
 
     @commands.group()
     async def todoset(self, ctx):
-        """Base settings command for customizing todo lists"""
-        pass  # Passing in subcommands isn't needed but okay
+        """Base settings command for customizing your todo list"""
+        pass
 
     @todoset.command()
     async def detailed(self, ctx, toggle: bool):
-        """Log popped todo reminders"""
-        await self._toggler(
-            ctx=ctx,
-            item="Detailed creation/removing",
-            key="detailed_pop",
-            toggle=toggle,
-        )
+        """Have a more detailed adding, removing, and completing message"""
+        if toggle:
+            ed = "enabled"
+        else:
+            ed = "disabled"
```

Comment:
We can probably merge this into a single line, as seen in the `usemd` function (L110). 

```suggestion
        ed = "enabled" if toggle else "disabled"
```